src/clj/commiteth/eth/multisig_wallet.clj

```diff
@@ -43,6 +43,7 @@
   [{:keys [owner internal-tx-id]}]
   {:pre [(string? owner) (string? internal-tx-id)]}
   (eth/execute {:internal-tx-id internal-tx-id
+                :type "deploy"
```

Comment:
Maybe instead of introducing another parameter here we could modify `internal-tx-id` to look something like this:

```clojure
[:deploy "1231455"]
```

Reply:
Great idea, have done so.